cfgov/cal/admin.py

```diff
@@ -28,21 +30,23 @@ def start_date(self, obj):
         return obj.dtstart
     start_date.short_description = 'Start Date'
 
+
 class CFPBPDFFileAdmin(admin.ModelAdmin):
     ordering = ['-id']
 
+
 class CFPBCalendarImportICSFileAdmin(admin.ModelAdmin):
 
     def changelist_view(self, request, extra_context=None):
         context = {}
         context.update(extra_context or {})
-        context.update({ 'icsfile': None,})
-        context.update({ 'calendar_record': None,})
-        context.update({ 'processed_events': None,})
+        context.update({'icsfile': None, })
```

Comment:
Weird, I get:

```
cfgov/cal/admin.py:43:40: E231 missing whitespace after ','
```

We can also eliminate the comma entirely (or ignore the error).